crypto/x509/x509_vfy.c

```diff
@@ -1754,119 +1754,70 @@ int X509_cmp_current_time(const ASN1_TIME *ctm)
 
 int X509_cmp_time(const ASN1_TIME *ctm, time_t *cmp_time)
 {
-    char *str;
+    static const size_t utctime_length = sizeof("YYMMDDHHMMSSZ") - 1;
+    static const size_t generalizedtime_length = sizeof("YYYYMMDDHHMMSSZ") - 1;
```

Comment:
No, our style guide doesn't say that constants should be uppercase. Left as is for now. Why would defines be better than typed constants?